terraform/modules/cloudfoundry/buckets.tf

```diff
@@ -39,3 +39,12 @@ module "logs-opensearch-archive" {
   acl              = "private"
   object_ownership = "ObjectWriter"
 }
+
+module "logs-opensearch-cf-audit-events" {
+  source           = "../s3_bucket/log_encrypted_bucket"
+  bucket           = "logs-opensearch-cf-audit-events-${var.stack_prefix}"
+  aws_partition    = var.aws_partition
+  expiration_days  = 372 # 31 days * 12 months = 372 days
```

Comment:
We need to keep things in S3 for at least 30 months, so ~930 days

Reply:
interesting, i thought because opensearch?